### ISSUE TITLE
Account verification - Block new forum accounts

### ISSUE THREAD

**Issue by Renato-Garcia** [Southclaws/cj#22]
We had a problem today with some toxic users, 'general' channel was set for only verified members (atm), but some of these users have new fresh accounts and got verified rank. The idea is block very new forum accounts, maybe 1 month be enough. It will prevent toxic users.

**Comment by JohnyMac** [comment]
I don't think this should be done at all, thats why mods and admins exists anyways..
However, if this would be implemented, I think it should be a reputation or posts requirement. Like have more than 10 REP or above 20 posts or something..

**Comment by Southclaws** [comment]
I like this suggestion, unfortunately admins aren't always around and abuse like this can suck if you're having a genuine conversation.

At the same time, I don't want to completely lock out new users. I think a (small) post count and account age is fine - just to stop people creating an account in the exact same day and spamming, I think a month is way too much.

**Comment by Crayder** [comment]
>unfortunately admins aren't always around and abuse like this can suck if you're having a genuine conversation

Much like the stuff that happened today when none of us were around to kick some kids out. They were spamming a bunch of bullshit and pissing @Hual off... xD

**Comment by WopsS** [comment]
Should we implement this and check the user page for:

* Join Date - don't allow then to talk if it joined that day.
* Posts - if he has 10 posts let him talk.
* Reputation - if he has more than 5 let him talk.

**Comment by Southclaws** [comment]
Yes that sounds fine, you can use `time.Parse` to turn the SA:MP forums date format into a timestamp.
